src/main/resources/org/jenkinsci/plugins/docker/workflow/Docker.groovy

```diff
@@ -159,8 +167,13 @@ class Docker implements Serializable {
         public void tag(String tagName = parsedId.tag, boolean force = true) {
             docker.node {
                 def taggedImageName = toQualifiedImageName(parsedId.userAndRepo + ':' + tagName)
-                // TODO as of 1.10.0 --force is deprecated; for 1.12+ do not try it even once
-                docker.script.sh "docker tag --force=${force} ${id} ${taggedImageName} || docker tag ${id} ${taggedImageName}"
+                // as of 1.10.0 --force is deprecated; for 1.12+ do not try it even once
+                if (docker.useTagForce) {
+                    docker.script.sh "docker tag --force=${force} ${id} ${taggedImageName} || docker tag ${id} ${taggedImageName}"
```

Comment:
So if we are on 1.9-, why keep the fallback to `tag` without `--force`?

Reply:
Because I was (foolishy) trying to make as few changes as possible.